gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/api/ApiResource.java

```diff
@@ -476,24 +474,23 @@ public Response verifyApiDeployment(@PathParam("apiId") String apiId) {
     @Produces(MediaType.APPLICATION_JSON)
     @Permissions({ @Permission(value = RolePermission.API_DEFINITION, acls = RolePermissionAction.READ) })
     public Response exportApiDefinition(
-        @Context HttpHeaders headers,
         @PathParam("apiId") String apiId,
         @QueryParam("excludeAdditionalData") Set<String> excludeAdditionalData
     ) {
-        if (excludeAdditionalData == null) {
-            excludeAdditionalData = Set.of();
-        }
-
-        ExportApiEntity exportApiEntity = apiImportExportService.exportApi(
-            GraviteeContext.getExecutionContext(),
-            apiId,
-            getAuthenticatedUser(),
-            excludeAdditionalData
-        );
+        var ctx = GraviteeContext.getExecutionContext();
+        var userDetails = getAuthenticatedUserDetails();
+        AuditActor auditActor = AuditActor
+            .builder()
+            .userId(userDetails.getUsername())
+            .userSource(userDetails.getSource())
+            .userSourceId(userDetails.getSourceId())
+            .build();
+        AuditInfo auditInfo = new AuditInfo(ctx.getOrganizationId(), ctx.getEnvironmentId(), auditActor);
```

Comment:
```suggestion
      AuditInfo auditInfo = getAuditInfo();
```

There is a `getAuditInfo()` method defined in `AbstractResource`